### DISCUSSION TITLE
Chrome Blocked all of my Github projects

### DISCUSSION THREAD

**Discussion by imtiazshawn** [community/community#39499] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

![image](https://user-images.githubusercontent.com/101446861/202847095-bf8ba9b5-da38-46a8-8e4b-a99dbe87dcbc.png)
There is an issue when I open any of my github projects, it shows "Deceptive site ahead
Attackers on imtiazshawn.github.io(My username)".

How can I fix it?

**Comment by subidit** [comment]
In your website repo enable https. 
Goto repo settings > pages > enforce HTTPS. 
It's at the bottom of the page. 

Hope it helps. Please mark as answered if it worked.

**Reply by imtiazshawn** [reply]
![image](https://user-images.githubusercontent.com/101446861/202860850-8fef637d-8a1e-4518-82cc-ba47e647f5e0.png)
I can't enable this.

**Comment by subidit** [comment]
You need to create a new **public** repo with name `imtiazshawn.github.io`. it's a special name repo that is required for enabling the GitHub webpage imtiazshawn.github.io. 

Just have a `index.html` in the main folder and then publish. 
For more info follow the steps here https://pages.github.com/

**Comment by airtower-luna** [comment, marked as answer]
That means your `github.io` domain landed on [Google's Safe Browsing blocklist](https://safebrowsing.google.com/). I've seen a lot of cases around this forum where that happened to people after they put a clone of some popular website (or at least its login page) up, which then counts as phishing. See the [Safe Browsing documentation](https://developers.google.com/search/docs/monitor-debug/security/social-engineering?hl=en) for how to deal with it, in short you can request a review, which obviously only makes sense after removing any such problematic content.

**Reply by imtiazshawn** [reply]
How can I fix it?

**Reply by airtower-luna** [reply]
Remove any such clone you might have, then check the Safe Browsing documentation (second link in my post above) for how to request a review.
